clar2wasm/Cargo.toml

```diff
@@ -21,8 +21,13 @@ stacks-common = { git="https://github.com/stacks-network/stacks-core", branch="f
 wat = "1.0.74"
 
 [features]
+default = ["test-clarity-v3"]
```

Comment:
nit: is this still necessary?

Reply:
No, since we have have `ClarityVersion::latest()` pointing to V3. Removed.